src/nft/hooks/usePermit2Approval.ts

```diff
@@ -3,12 +3,31 @@ import { InterfaceEventName } from '@uniswap/analytics-events'
 import { CurrencyAmount, Token } from '@uniswap/sdk-core'
 import { UNIVERSAL_ROUTER_ADDRESS } from '@uniswap/universal-router-sdk'
 import { useWeb3React } from '@web3-react/core'
+import { usePwatNewContractEnabled } from 'featureFlags/flags/pwatNewContract'
 import usePermit2Allowance, { AllowanceState } from 'hooks/usePermit2Allowance'
 import { useCallback, useMemo, useState } from 'react'
 import invariant from 'tiny-invariant'
 
-// TODO: This should be removed when the sdk is updated to include the new UR address
```

Comment:
q: is the sdk upgrade still blocked by BE updates?

Reply:
yeah but now we have a like a feature flag setup and stuff so I think its a little more clear whats going on vs a hardcoded string